File: app/src/main/java/social/entourage/android/entourage/create/CreateEntourageListener.java
package social.entourage.android.entourage.create;

import social.entourage.android.entourage.category.EntourageCategory;

/**
 * Created by mihaiionescu on 18/05/2017.
 */

public interface CreateEntourageListener {

    void onTitleChanged(String title);
    void onDescriptionChanged(String description);
    void onCategoryChosen(EntourageCategory category);

}

File: app/src/main/java/social/entourage/android/entourage/create/CreateEntourageListener.kt
package social.entourage.android.entourage.create

import social.entourage.android.entourage.category.EntourageCategory

/**
 * Created by mihaiionescu on 18/05/2017.
 */
interface CreateEntourageListener {
    fun onTitleChanged(title: String)
    fun onPrivacyChanged(privacy: Boolean)
    fun onDescriptionChanged(description: String)
    fun onCategoryChosen(category: EntourageCategory)
    fun onPhotoEventAdded(portrait_url:String?,landscape_url:String?)
}